yorkie/src/main/kotlin/dev/yorkie/document/operation/OperationInfo.kt

```diff
@@ -0,0 +1,60 @@
+package dev.yorkie.document.operation
+
+import dev.yorkie.document.crdt.TextWithAttributes
+import dev.yorkie.document.time.TimeTicket
+
+/**
+ * [OperationInfo] represents the information of an operation.
+ * It is used to inform to the user what kind of operation was executed.
+ */
+public sealed class OperationInfo {
+
+    public abstract var path: String
+
+    internal var executedAt: TimeTicket = TimeTicket.InitialTimeTicket
```

Comment:
[js implementation link](https://github.com/yorkie-team/yorkie-js-sdk/blob/9f6e51b5352da2537389fb776c7997af518214cb/src/document/operation/operation.ts#L114)
I changed the property name from `element` to `executedAt`  becauase`element` sounds more appropriate for something like `CrdtElement` or `JsonElement`, while `executedAt` has been used as the property name for `TimeTicket`until now.
(cc. @chacha912 )

Reply:
JS-SDK has been updated to remove `InternalOpInfo`. Could you please review the changes and make the necessary updates accordingly?
https://github.com/yorkie-team/yorkie-js-sdk/pull/566